crates/neon/src/types_impl/extract/private.rs

```diff
@@ -35,6 +36,8 @@ impl Sealed for &str {}
 
 impl<'cx, V: Value> Sealed for Handle<'cx, V> {}
 
+impl<O: Object> Sealed for Root<O> {}
```

Comment:
Is this here so that it passes the preconditions for a `Root<O>` to be able to implement `TryFromJs`?

Reply:
Update: yes, now that I see the diff in try_from_js.rs, my understanding was right.